app/serializers/solidus_shipwire/address_serializer.rb

```diff
@@ -7,7 +7,7 @@ class AddressSerializer < ActiveModel::Serializer
     attribute :state_name, key: :state
     attribute :zipcode,    key: :postalCode
 
-    attribute(:name)    { "#{object.firstname} #{object.lastname}" }
+    attribute(:name)    { object.name }
```

Comment:
You should use `SolidusSupport.combined_first_and_last_name_in_address?` to conditionally use `name` or `fullname`, based on the solidus version used.